packages/assistant-stream/src/core/modules/assistant-stream.ts

```
@@ -0,0 +1,230 @@
+      | {
+          toolCallId?: string;
+          toolName: string;
+          args?: Record<string, unknown>;
+          result?: unknown;
+        },
+  ): ToolCallStreamController {
+    const opt = typeof options === "string" ? { toolName: options } : options;
+    const toolName = opt.toolName;
+    const toolCallId = opt.toolCallId ?? generateId();
+
+    let [stream, controller] = createToolCallStreamController();
```

Biome (1.9.4)

[error] This let declares some variables that are only assigned once.

'stream' is never reassigned.

'controller' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)